openpype/lib/transcoding.py

```diff
@@ -191,6 +194,14 @@ def parse_oiio_xml_output(xml_string, logger=None):
     if not xml_string:
         return output
 
+    # Fix values with ampresand (lazy fix)
+    # - oiiotool exports invalid xml which ElementTree can't handle
+    #   e.g. "&#01;"
```

Comment:
I would just add explicit warning about what it does and point it to this discussions.

```suggestion
    #   e.g. "&#01;"
    # WARNING: this will affect even valid character entities. If you need those values correctly, this must
    # take care of valid character ranges. See https://github.com/pypeclub/OpenPype/pull/2729
```